releasenotes/notes/update-prompt-annotation-0fa90edf6829fe1d.yaml

```diff
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    LLM Observability: This introduces prompt annotation for ``LLMObs.llm`` and a new ``prompt_context`` annotation context to add prompts.
+upgrade:
```

Comment:
This category is usually for deprecation/major version bump notices, we can move this to features instead